tmk_core/protocol/vusb/vusb.c

```diff
@@ -71,6 +79,52 @@ void vusb_transfer_keyboard(void) {
     }
 }
 
+/*------------------------------------------------------------------*
+ * RAW HID
+ *------------------------------------------------------------------*/
+#ifdef RAW_ENABLE
+#    define RAW_INPUT_SIZE (32)
+#    define RAW_OUTPUT_SIZE (32)
+
+static uint8_t raw_output_buffer[RAW_OUTPUT_SIZE];
+static uint8_t raw_output_recieved_bytes = 0;
```

Comment:
```suggestion
static uint8_t raw_output_received_bytes = 0;
```
;)

Reply:
I missed this!
But this is corrected
Thanks a lot :)

Reply:
I meant that this is actually the correct spelling, and you should replace the rest of the instances of `raw_output_recieved_bytes`.

Reply:
I What I wanted to say is that this is already fixed.
I'm sorry for my poor English.

Reply:
That commit changes it back to the wrong spelling.

Reply:
ahhhhh. I made misstake. thanks.